src/react-components/ui-root.js

```diff
@@ -817,7 +818,10 @@ class UIRoot extends Component {
 
   renderEntryStartPanel = () => {
     const { hasAcceptedProfile, hasChangedName } = this.props.store.state.activity;
-    const promptForNameAndAvatarBeforeEntry = this.props.hubIsBound ? !hasAcceptedProfile : !hasChangedName;
+    const isLockedDownDemo = isLockedDownDemoRoom(this.props.hub.hub_id);
```

Comment:
My thinking is that it is still important to be able to test your speaker output in the entry flow. I might ask Jacob if this is truly a priority, but I anticipate skipping the audio step will lead to difficulty troubleshooting audio experiences.

Reply:
ok, I wasn't sure about what was the expected behavior there. In that case it might be a good idea to disable the audio input setup. That will disable the mic permissions trigger. We can do that here:

https://github.com/mozilla/hubs/blob/7518e63052521b07c302f42bf7c16121f276ddfc/src/react-components/room/MicSetupModalContainer.js#L38